gateway-api/src/lib.rs

```diff
@@ -313,6 +314,26 @@ pub trait GatewayApi {
         params: TypedBody<GetRotBootInfoParams>,
     ) -> Result<HttpResponseOk<RotState>, HttpError>;
 
+    /// Get the number of task dumps present on an SP
+    #[endpoint {
+        method = GET,
+        path = "/sp/{type}/{slot}/task-dump",
+    }]
+    async fn sp_host_task_dump_count(
```

Comment:
Nit - is `host` right in these two function names? (To me that implies "dump from the host OS", which is not right, but maybe I'm thinking of it incorrectly?)

Reply:
Yeah that's a good call, I've dropped `host` from the names to just `sp_task_dump_*`.